syntax = "proto3";

package bithub;

// Define a message for PR Description Request
message PRDescriptionRequest {
    string repository_url = 1;
    string branch_name = 2;
}

// Define a message for PR Description Response
message PRDescriptionResponse {
    string description = 1;
}

// Define the AI Assistant Service
service AIAssistant {
    rpc GeneratePRDescription (PRDescriptionRequest) returns (PRDescriptionResponse);
    // Other functions defined similarly
}
